python/darpinstances/instance.py

```diff
@@ -179,6 +178,51 @@ def load_vehicles(vehicles_path: str) -> List[Vehicle]:
     return vehicles
 
 
+equipment_list = [
+    "NONE",
+    "STANDARD_SEAT",
+    "WHEELCHAIR",
+    "ELECTRIC_WHEELCHAIR",
+    "MEDICAL_STROLLER",
+]
+
+
+def get_equipment_int(str):
+    if str in equipment_list:
+        return equipment_list.index(str)
+    else:
+        return -1
+
+
+def load_vehicles_from_json(vehicles_path: str) -> List[Vehicle]:
+    veh_data = darpinstances.inout.load_json(vehicles_path)
+    vehicles = []
+    list = veh_data["vehicles"]
+    for index, veh in enumerate(list):
+        equipment = []
+        if "slots" in veh:
+            for item in veh["slots"]:
+                count = int(item["count"])
+                type = get_equipment_int(item["type"])
+                for n in range(count):
+                    equipment.append(type)
+        elif "configurations" in veh:
+            counters = [0] * len(equipment_list)
```

Comment:
I think you lose the information about configurations this way

Reply:
From our conversations it was defined that the json would contain either slots or configurations not both